app/_src/mesh/features/openshift-quickstart.md

```diff
@@ -0,0 +1,184 @@
+
+## Install Kong Mesh in Standalone Mode
+
+1. Create the `kong-mesh-system` namespace:
+    ```bash
+    kubectl create namespace kong-mesh-system
+    ```
+
+1. Create the image pull secret:
+    ```bash
+    kubectl create secret docker-registry rh-registry-secret -n kong-mesh-system \
+        --docker-server=registry.connect.redhat.com \
+        --docker-username=<username> \
+        --docker-password=<password> \
+        --docker-email=<email>
+    ```
+
+1. Add nonroot-v2 to job service accounts:
+    ```bash
+    oc adm policy add-scc-to-user nonroot-v2 system:serviceaccount:kong-mesh-system:kong-mesh-install-crds
+    oc adm policy add-scc-to-user nonroot-v2 system:serviceaccount:kong-mesh-system:kong-mesh-patch-ns-job 
+    oc adm policy add-scc-to-user nonroot-v2 system:serviceaccount:kong-mesh-system:kong-mesh-pre-delete-job
+    ```
+
+1. Grab the latest helm chart:
+    ```bash
+    helm repo add kong-mesh https://kong.github.io/kong-mesh-charts
+    ```
+
+1. Install Kong Mesh:
+    ```bash
+    helm upgrade -i kong-mesh kong-mesh/kong-mesh --version 2.2.0 -f kong-mesh/values.yaml -n kong-mesh-system
+    ```
+
+1. Open a second terminal and port-forward to reach the mesh GUI:
+    ```bash
+    kubectl port-forward svc/kong-mesh-control-plane -n kong-mesh-system 5681:5681
+    ```
+
+    You should be able to reach the Kong-Mesh UI at `http://localhost:5681/gui`.
+
+
+1. Last, do some prep work for the sidecar itself so sidecars will startup successfully. Apply the kong-mesh-sidecar scc and corresponding container patches:
+    ```bash
+    kubectl create -f kong-mesh/kong-mesh-sidecar-scc.yaml
+    kubectl apply -f kong-mesh/container-patch.yaml 
+    ```
+
+## Deploy KIC on Mesh
+
+1. First we're going to deploy KIC as our ingress controller:
+    ```bash
+    kubectl create namespace kong 
+    kubectl label namespace kong kuma.io/sidecar-injection=enabled
+    oc adm policy add-scc-to-user kong-mesh-sidecar system:serviceaccount:kong:kong-kong
+    ```
+
+1. Grab the latest chart:
+    ```bash
+    helm repo add kong https://charts.konghq.com
+    helm repo update
+    ```
+
+1. Then run the install:
+    ```bash
+    helm install kong kong/kong -n kong \
+    --set ingressController.installCRDs=false \
+    --set podAnnotations."kuma\.io/mesh"=default \
+    --set podAnnotations."kuma\.io/gateway"=enabled
+    export PROXY_IP=$(kubectl get -o jsonpath="{.status.loadBalancer.ingress[0].hostname}" service -n kong kong-kong-proxy)
+    curl -i $PROXY_IP
+    ```
+
+    The expected output is:
+    ```json
+    HTTP/1.1 404 Not Found
+    Date: Tue, 02 May 2023 15:40:05 GMT
+    Content-Type: application/json; charset=utf-8
+    Connection: keep-alive
+    Content-Length: 48
+    X-Kong-Response-Latency: 0
+    Server: kong/3.2.2
+
+    {"message":"no Route matched with those values"}%
+    ```
+
+## Deploy Bookinfo
```

Comment:
What does this app do?

Reply:
This is an opensourced istio example that comes with 4 microservices, different versions to test out mesh capabilities. 

It will display information about a book. 

More details can be found here: https://istio.io/latest/docs/examples/bookinfo/

Reply:
yeah that's a good idea, we just need to get off of evaluation mode with a license to have enough sidecars for the kuma app. I can do a quick test on openshift to make sure there are no other issues.

Reply:
Thanks!

Reply:
I am concerned the postgres pod of the kuma demo is going to fail on openshift without some modification. Is there another usable demo?